contracts/src/ligo.ts

```diff
@@ -57,26 +56,26 @@ export async function loadFile(fileName: string): Promise<string> {
   );
 }
 
-export async function compileContract(
+export function compileContract(
   env: LigoEnv,
   srcFile: string,
   main: string,
   dstFile: string
 ): Promise<void> {
   const src = env.srcFilePath(srcFile);
   const out = env.outFilePath(dstFile);
-  await compileContractImpl(env.cwd, src, main, out);
+  return compileContractImpl(env.cwd, src, main, out);
 }
 
-async function compileContractImpl(
+function compileContractImpl(
   cwd: string,
   srcFilePath: string,
   main: string,
   dstFilePath: string
 ): Promise<void> {
   // const cmd = `ligo compile-contract ${srcFilePath} ${main} --output=${dstFilePath}`;
-  const cmd = `docker run --rm -v $PWD:$PWD -w $PWD ligolang/ligo:0.11.0 compile-contract ${srcFilePath} ${main} --output=${dstFilePath}  && rm bisect*.coverage`;
-  await runCmd(cwd, cmd);
+  const cmd = `docker run --rm -v $PWD:$PWD -w $PWD ligolang/ligo:0.11.0 compile-contract ${srcFilePath} ${main} --output=${dstFilePath}  && rm -f bisect*.coverage`;
```

Comment:
Adding `-f` to the `rm bisect*.coverage` command was necessary to suppress warnings in the event that one of the parallel processes already deleted files matching the `bisect*.coverage` pattern.

Reply:
:+1:

Reply:
LGTM :+1: